adaptive_breakpoints/example/lib/main.dart

```diff
@@ -16,43 +16,48 @@ class AdaptiveBreakpointsExample extends StatelessWidget {
       home: Scaffold(
         body: Column(
           children: [
-            AdaptiveContainer(
-              adaptiveConstraints: AdaptiveConstraints(xs: true),
+            _AdaptiveContainer(
+              adaptiveConstraints:
+              _AdaptiveConstraints(xs: true),
```

Comment:
The format on this is odd, it should go on the line above right? Same problem throughout this example

Reply:
Done